Description: use pkg-config for glib includes
 The hard coded list of include locations for glib is incompatible
 with multiarch. Use pkg-config instead.
Author: Peter Green <plugwash@p10link.net>
Bug-Debian: http://bugs.debian.org/652162

Index: pidgin-facebookchat-1.69/Makefile
===================================================================
--- pidgin-facebookchat-1.69.orig/Makefile	2011-12-31 22:18:01.000000000 +0000
+++ pidgin-facebookchat-1.69/Makefile	2011-12-31 22:18:39.000000000 +0000
@@ -11,7 +11,7 @@
 MACPORT_COMPILER = i686-apple-darwin10-gcc-4.0.1
 
 LIBPURPLE_CFLAGS = -I/usr/include/libpurple -I/usr/local/include/libpurple -DPURPLE_PLUGINS -DENABLE_NLS -DHAVE_ZLIB
-GLIB_CFLAGS = -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include -I/usr/local/include/glib-2.0 -I/usr/local/lib/glib-2.0/include -I/usr/local/include -I/usr/include/json-glib-1.0 -ljson-glib-1.0
+GLIB_CFLAGS := $(shell pkg-config --cflags glib-2.0) $(shell pkg-config --cflags json-glib-1.0)
 WIN32_DEV_DIR = /root/pidgin/win32-dev
 WIN32_PIDGIN_DIR = /root/pidgin/pidgin-2.3.0_win32
 WIN32_CFLAGS = -I${WIN32_DEV_DIR}/gtk_2_0/include/glib-2.0 -I${WIN32_PIDGIN_DIR}/libpurple/win32 -I${WIN32_DEV_DIR}/gtk_2_0/include -I${WIN32_DEV_DIR}/gtk_2_0/include/glib-2.0 -I${WIN32_DEV_DIR}/gtk_2_0/lib/glib-2.0/include -I/usr/include/json-glib-1.0 -Wno-format
